src/ingestion_data.py

```diff
@@ -4,6 +4,15 @@
 
 from src.graph_model import (Version, EventType, SecurityEvent, FeedBackType, StatusType, EcosystemType)
 from src.parse_datetime import from_date_str, get_date, get_year, get_yearmonth
+from src.config import MAX_STRING_LENGTH
+
+CVE_REGULAR_EXPRESSION = r"CVE-\d{4}-\d{4,}"
```

Comment:
This should be private constant.
```suggestion
_CVE_REGULAR_EXPRESSION = r"CVE-\d{4}-\d{4,}"
```

Reply:
done